draft-ietf-ppm-dap.md

```diff
@@ -1006,9 +1006,14 @@ the following parameters associated with it:
   are grouped into batches and the properties that all batches for this task
   must have. The party MUST NOT configure the task if it does not recognize the
   batch mode.
+* `task_start`: The start timestamp of a task. Clients MAY opt out of the task
```

Comment:
"opt out" is a taskprov term that we probably shouldn't overload. I'd suggest "don't upload reports" instead of "opt out". Likewise below.

Reply:
I added `MAY not upload its report`. Let me know if that's the right use of MAY..